nav2_route/src/goal_intent_extractor.cpp

```diff
@@ -122,6 +138,10 @@ GoalIntentExtractor::findStartandGoal(const std::shared_ptr<const GoalT> goal)
             "Could not determine node closest to start or goal pose requested!");
   }
 
+  if (!isNodeValid(start_route, start_))
+  {
+    RCLCPP_INFO(logger_, "Invalid Starting Route index");
```

Comment:
Should this not be throwing an exception of some kind of its truly an error for the action to reject a request? More context on none of the nearest starting points are reachable would be good for the message instead.